rct229/rules/section5/section5rule15.py

```diff
@@ -71,6 +71,11 @@ def __init__(self):
                         "$": ["construction"],
                         "construction": ["f_factor"],
                     },
+                    manual_check_required_msg="Zone has both residential and non-residential spaces and the "
+                                              "construction requirements for slab-on-grade floor are different. "
+                                              "Verify construction is modeled correctly.",
+                    fail_msg='Baseline slab F-factor is not as expected for slabs that are less than 24" below grade. '
```

Comment:
It looks like you need to run Black on this. I expect it will replace the single quotes with double quotes.

Reply:
Ran Black did not change the single quotes.
Should be safe I guess...